### ISSUE TITLE
Re-explore path data blocked by temporary fields or creatures

### ISSUE THREAD

**Issue by LeeKunTibia** [tibiamaps/tibia-map-data#102]
A few years ago CipSoft stopped saving fields/creatures on the Path files, but many areas in TibiaMaps are still blocked by these fields which makes map-clicking impossible. The aim of this issue is to track down these areas so we can reupload their path files and avoid opening a new issue for every area detected with this problem. 

- ~Calassa~ (fixed with #61)
- ~Emperor's Castle and Hive~ (fixed with #81) 
- Grounds of Destruction (Jugger Seal)
- Path to Hellgate from Ab'dendriel (specially the narrow corridor with skeletons)

**Comment by LeeKunTibia** [comment]
Grounds of Destruction (Jugger Seal) + Undeath (Undead Dragon) + Hell Hub:

[Ferumbras Path.zip](https://github.com/tibiamaps/tibia-map-data/files/5369322/Ferumbras.Path.zip)

**Comment by mathiasbynens** [comment]
> Grounds of Destruction (Jugger Seal) + Undeath (Undead Dragon) + Hell Hub:

Thanks! Merged and committed. Should be live in the downloads in ~20 minutes or so.

**Comment by LeeKunTibia** [comment]
Turns out [Bounac](https://tibiamaps.io/map#32416,32487,7:0)'s original data was missing all Path files, so here they are:

[Bounac Path.zip](https://github.com/tibiamaps/tibia-map-data/files/5382037/Bounac.Path.zip)

**Comment by LeeKunTibia** [comment]
Roshamuul Prison:

[Roshamuul Prison.zip](https://github.com/tibiamaps/tibia-map-data/files/5402753/Roshamuul.Prison.zip)

**Comment by LeeKunTibia** [comment]
Warzones 1-3 have the same issue. I mapped the creatures areas and the Warzone 2 boss room, but Warzones 1 and 3 will have to wait (but are less important as well). 

[Warzones 1-3.zip](https://github.com/tibiamaps/tibia-map-data/files/5431656/Warzones.1-3.zip)

**Comment by LeeKunTibia** [comment]
Since the Hellgate path covers lots of floors, I'm only going to send the floor I'm sure has a problem right now. After it's synced I'll update my maps and try again in game to check if any other floor needs fixing. 

[Minimap_WaypointCost_32512_31488_13.zip](https://github.com/tibiamaps/tibia-map-data/files/5560940/Minimap_WaypointCost_32512_31488_13.zip)

**Comment by mathiasbynens** [comment]
@LeeKunTibia Merged. If you wanna continue testing, the updated maps should be available for downloads soon!

**Comment by LeeKunTibia** [comment]
Found another blocked passage on Hellgate, floor 12 this time. 

[Minimap_WaypointCost_32512_31488_12.zip](https://github.com/tibiamaps/tibia-map-data/files/5604387/Minimap_WaypointCost_32512_31488_12.zip)

**Comment by alfredomtx** [comment]
The way to Mintwallin, going down ancient temple, there is a small poison field passage, in Tibiamap's Cost files it is not considered walkable, here is the new one:

[Minimap_WaypointCost_32256_32000_12.zip](https://github.com/tibiamaps/tibia-map-data/files/5844411/Minimap_WaypointCost_32256_32000_12.zip)

**Comment by alfredomtx** [comment]
Kazordoon path to Pydar(bless) fire fields [here] (https://tibiamaps.io/map#32631,31890,11:2)

[Uploading Minimap_WaypointCost_32512_31744_11.zip…]()

**Comment by mathiasbynens** [comment]
@alfredomtx I think you may have posted that comment before the file finished uploading — the link doesn't work :( Would you mind trying again? Thanks!

**Comment by alfredomtx** [comment]
> @alfredomtx I think you may have posted that last comment before the file finished uploading — the link doesn't work :( Would you mind trying again? Thanks!

Sure!

Here it is:
[Minimap_WaypointCost_32512_31744_11.zip](https://github.com/tibiamaps/tibia-map-data/files/5851448/Minimap_WaypointCost_32512_31744_11.zip)

**Comment by mathiasbynens** [comment]
@alfredomtx Thanks! Updated.

**Comment by LeeKunTibia** [comment]
Upper Roshamuul ground floor:

[Minimap_WaypointCost_33536_32256_7.zip](https://github.com/tibiamaps/tibia-map-data/files/5901063/Minimap_WaypointCost_33536_32256_7.zip)

**Comment by LeeKunTibia** [comment]
Hero of Rathleton (Jaccus Maxxen's Dungeon) and Vandura Rotworms:
[Hero and Vandura.zip](https://github.com/tibiamaps/tibia-map-data/files/6665843/Hero.and.Vandura.zip)

**Comment by LeeKunTibia** [comment]
And here we go again:

Otherworld: [Otherworld.zip](https://github.com/tibiamaps/tibia-map-data/files/6689231/Otherworld.zip)

Oramond Dungeon and Furies: [Oramond Catacombs and Furies.zip](https://github.com/tibiamaps/tibia-map-data/files/6689234/Oramond.Catacombs.and.Furies.zip)

Grounds of Fire, Deceit, Destruction and Plague: [Grounds.zip](https://github.com/tibiamaps/tibia-map-data/files/6689238/Grounds.zip)

**Comment by mathiasbynens** [comment]
@LeeKunTibia Thanks! Added.

**Comment by LeeKunTibia** [comment]
Here's a new batch. To exemplify and register how "bad" some of these areas are/were, here's a comparison:

![image](https://user-images.githubusercontent.com/31053487/123476994-d303f600-d5d3-11eb-9b84-ad2a26a90dad.png)

[Grounds of Damnation.zip](https://github.com/tibiamaps/tibia-map-data/files/6718442/Grounds.of.Damnation.zip)

[Halls of Ascension.zip](https://github.com/tibiamaps/tibia-map-data/files/6718443/Halls.of.Ascension.zip)

[PoI e Inq.zip](https://github.com/tibiamaps/tibia-map-data/files/6718444/PoI.e.Inq.zip)

[Roshamuul Prison 2.zip](https://github.com/tibiamaps/tibia-map-data/files/6718445/Roshamuul.Prison.2.zip)

**Comment by mathiasbynens** [comment]
@LeeKunTibia Incredible! Nice work.

**Comment by LeeKunTibia** [comment]
Sorry, I messed up this one, still had a blocked corridor to the west.

[Minimap_WaypointCost_33024_30976_12.zip](https://github.com/tibiamaps/tibia-map-data/files/8056726/Minimap_WaypointCost_33024_30976_12.zip)

**Comment by mathiasbynens** [comment]
@LeeKunTibia Thanks, done!

**Comment by LeeKunTibia** [comment]
Oramond Demons. 

[Minimap_WaypointCost_33280_31744_10.zip](https://github.com/tibiamaps/tibia-map-data/files/8459771/Minimap_WaypointCost_33280_31744_10.zip)

**Comment by LeeKunTibia** [comment]
Lower Spike: [Lower Spike.zip](https://github.com/tibiamaps/tibia-map-data/files/9033309/Lower.Spike.zip)
 
(Apparently the Oramond Demons above wasn't synced yet)

**Comment by mathiasbynens** [comment]
Done!

**Comment by LeeKunTibia** [comment]
Several Oramond Underground Areas (both Glooth Factories and Raids):
[Oramond Underground.zip](https://github.com/tibiamaps/tibia-map-data/files/9033364/Oramond.Underground.zip)

**Comment by LeeKunTibia** [comment]
Been to Demon today and some paths of MoLS were blocked by creatures. This isn't a full fix but at least it allows map-clicking there easier. 

[MoLS.zip](https://github.com/tibiamaps/tibia-map-data/files/9438622/MoLS.zip)

**Comment by mathiasbynens** [comment]
Added!

**Comment by mathiasbynens** [comment]
Cyclopolis:

```
33024_31488_8
33024_31488_9
33024_31488_10
33024_31488_11
33280_31488_9
33280_31488_10
33280_31488_11
33280_31488_12
33280_31488_13
```

**Comment by mathiasbynens** [comment]
Way to Groam: `32512_32000_9` (We should use the pathfinding data from when the event is active to enable map-clicking down the stairs.)

**Comment by LeeKunTibia** [comment]
Zzaion: 
[Zzaion.zip](https://github.com/tibiamaps/tibia-map-data/files/10033269/Zzaion.zip)

**Comment by mathiasbynens** [comment]
> Zzaion: [Zzaion.zip](https://github.com/tibiamaps/tibia-map-data/files/10033269/Zzaion.zip)

Thanks; added!

**Comment by LeeKunTibia** [comment]
I managed to explore the other 2 solo Prison Bosses, unfortunately did not have a team despite having the Key on TS to enter Drazzak's.

[Minimap_WaypointCost_33536_32256_12.zip](https://github.com/tibiamaps/tibia-map-data/files/10072205/Minimap_WaypointCost_33536_32256_12.zip)

**Comment by mathiasbynens** [comment]
Merged!

**Comment by mathiasbynens** [comment]
Zorvorax teleport floor:

- `33024_31488_11`
- `32768_31488_11`

**Comment by mathiasbynens** [comment]
I _think_ the Heart of Destruction accessible via Zao still suffers from this problem, in particular the entire outer-most ring. Check out this location and press `P` in the map viewer to see the pathfinding data: https://tibiamaps.io/map#32114,31399,13:2

**Comment by mathiasbynens** [comment]
One more floor in Hellgate on the way to Draconia: `32768_31488_10`
https://tibiamaps.io/map#32806,31578,10:2

[`Minimap_WaypointCost_32768_31488_10.zip`](https://github.com/tibiamaps/tibia-map-data/files/11436226/Minimap_WaypointCost_32768_31488_10.zip)

**Comment by mathiasbynens** [comment]
Found some more blocked tiles on the way to Feru seals:
[`feru-seals.zip`](https://github.com/tibiamaps/tibia-map-data/files/11581135/feru-seals.zip)

**Comment by LeeKunTibia** [comment]
Fenrock Dragon Lair

[Minimap_WaypointCost_32512_31232_15.zip](https://github.com/tibiamaps/tibia-map-data/files/12930028/Minimap_WaypointCost_32512_31232_15.zip)

**Comment by mathiasbynens** [comment]
> Fenrock Dragon Lair
> 
> [Minimap_WaypointCost_32512_31232_15.zip](https://github.com/tibiamaps/tibia-map-data/files/12930028/Minimap_WaypointCost_32512_31232_15.zip)

Merged, thanks!

**Comment by muriloricci** [comment]
[Dark Thais small changes.zip](https://github.com/user-attachments/files/16246322/Dark.Thais.small.changes.zip)

We tried to hide in a corner to kill a fiendish in Dark Thais but the corner didn't exist. :P

**Comment by muriloricci** [comment]
[Minimap_WaypointCost_32256_31488_9.zip](https://github.com/user-attachments/files/16536204/Minimap_WaypointCost_32256_31488_9.zip)

I've noticed that every time I clicked one of the marks to enter MOLS my character would just refuse to go, so I noticed so waypoint cost data is missing here.

**Comment by muriloricci** [comment]
Zao's Emperor's Castle is always a frustrating because the pathing is all broken. Hopefully this should fix things forever. :P
[Emperor's Castle waypoint costs.zip](https://github.com/user-attachments/files/16648881/Emperor.s.Castle.waypoint.costs.zip)

**Comment by muriloricci** [comment]
[Bounac + Podzilla task.zip](https://github.com/user-attachments/files/16917125/Bounac.%2B.Podzilla.task.zip)

**Comment by mathiasbynens** [comment]
[`improved-gray-island-paths.zip`](https://github.com/user-attachments/files/18141997/improved-gray-island-paths.zip)

**Comment by muriloricci** [comment]
Some spots I've noticed needing pathing fix (don't know if they were listed before, I'm adding as I spot the issues):
- Goblins in Edron (surface and some spots underground)
- Edron HoD
- Dragon Lords at POI
- POI levers area
- Mother of Scarabs Lair
- Elemental Spheres Quest access areas (energy, fire and earth)
- Vulcongra area in WZ123
- Way to lizard tower in Zao (through brimstone bugs, etc - forgot the name now)
- Some spots in the Banshee Quest (minor spots but decent amount of minor spots)
- Hero or Rathleton bossfight rooms
- Tombs in Ankrahmun
- Mines in Svargrond (ice portal)
- Fury Gate
- Ghastly Dragons at Zao (deep underground ones)
- One of the rooms in Demona

**Comment by muriloricci** [comment]
[Map Fixes - Dark Thais, Goblin Edron, WZ123 and Edron HoD.zip](https://github.com/user-attachments/files/19341085/Map.Fixes.-.Dark.Thais.Goblin.Edron.WZ123.and.Edron.HoD.zip)

**Comment by muriloricci** [comment]
[Map Fixes - DLs POI, POI levers, Mother of Scarabs Lair, Hero of Rathleton.zip](https://github.com/user-attachments/files/19364105/Map.Fixes.-.DLs.POI.POI.levers.Mother.of.Scarabs.Lair.Hero.of.Rathleton.zip)
